website/docs/docs/collaborate/manage-access/enterprise-permissions.md

```diff
@@ -44,6 +44,26 @@ Account Admins have unrestricted access to dbt Cloud accounts. Users with Accoun
 - Use the IDE
 - Run and cancel jobs
 
+### Project Creator
+- **Has permissions on:** Authorized projects, account-level settings
+- **License restrictions:** must have a developer license
+
+Project Creators have write and read only access to dbt Cloud accounts, but do not have permissions to modify SSO settings and account integrations. 
+
+Users with Project Creator permissions can:
+
+- View Account Settings
+- View and modify project users
+- Create, delete and modify all projects in an account
+- Create, delete, and modify Repositories
+- Create, delete, and modify Connections
+- Create, delete, and modify Environments
+- Create, delete, and modify Jobs
+- Use the IDE
+- Run and cancel jobs
+- View Groups
+- View notification settings
```

Comment:
For my own clarification, how are we choosing what to capitalize after the first word?  Are the capitalized words all menu items or feature selections in the product?  If that's the case should "notification settings" be capitalized?

Reply:
ah yes good callout @matthewshaver ! any UI items were capitalized to match the UI but i noticed the other permissions didn't capitalize 'Notification Settings' so i wasn't sure if it should be lowercase. But I've double-checked the UI and have changed it and also changed the other permissions too.